libs/core/src/lib/calendar/calendar.component.ts

```diff
@@ -232,6 +232,18 @@ export class CalendarComponent<D> implements OnInit, ControlValueAccessor, Valid
     @Input()
     disableFunction: (date: D) => boolean = () => false;
 
+    /**
+     * Function used to disable previous button in the calendar header.
+     */
+    @Input()
+    previousButtonDisableFunction: (date?: D) => boolean = () => false;
```

Comment:
I don't think default value is needed, you should just handle the case provided value is not a function in `calnedar-header`